### DISCUSSION TITLE
AutoLogin with start chromium kiosk

### DISCUSSION THREAD

**Discussion by kalwinskidawid** [minios-linux/minios-live#50] [Q&A] [answered]
Hey, 
Is it possible to change autologin user after installed miniOS? 
I'd like to use just [chromium-kiosk](https://github.com/Salamek/chromium-kiosk)

**Comment by FershoUno** [comment, marked as answer]
Hello, this is possible, you need to change the "autologin" option in the minios configuration file

https://github.com/minios-linux/minios-live/wiki/Configuration-file
